diagnostics-app/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "diagnostics-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~12.0.2",
```

Comment:
Does package.json support comments? I'd like to add a brief comment to each dependency explaining what is it. WDYT?